app/javascript/app/phone-internationalization.js

```diff
@@ -72,12 +72,16 @@ const updateOTPDeliveryMethods = () => {
     phoneLabel.classList.add('btn-disabled');
     smsRadio.click();
     deliveryMethodHint.innerText = warningMessage;
-    optPhoneLabelInfo.innerText = I18n.t('devise.two_factor_authentication.otp_phone_label_info_modile_only');
+    if (optPhoneLabelInfo) {
+      optPhoneLabelInfo.innerText = I18n.t('devise.two_factor_authentication.otp_phone_label_info');
+    }
   } else {
     phoneRadio.disabled = false;
     phoneLabel.classList.remove('btn-disabled');
     deliveryMethodHint.innerText = I18n.t('devise.two_factor_authentication.otp_delivery_preference.instruction');
-    optPhoneLabelInfo.innerText = I18n.t('devise.two_factor_authentication.otp_phone_label_info');
+    if (optPhoneLabelInfo) {
+      optPhoneLabelInfo.innerText = I18n.t('devise.two_factor_authentication.otp_phone_label_info_mobile_only');
+    }
   }
 };
```

Comment:
For the Code Climate complexity issue here, would it make sense to move some of the constants out of the `updateOTPDeliveryMethods` function? For example, I noticed that these 2 lines are defined in multiple places:
```js
const phoneInput = document.querySelector('[data-international-phone-form] .phone');
const phone = phoneInput.value;
```
Also, why are we now checking `if (optPhoneLabelInfo)` whereas we didn't before, and the other elements don't have conditionals?

Reply:
I see that you removed the span that had the `otp_phone_label_info` id, which explains why you are now checking for its presence. Why was it removed? Previously, that span was there to provide additional help info to the user, such as saying that if they enter a landline, they should select the "Phone call" option. Was that removed on purpose on this new screen?

Reply:
Hey @monfresh, good catch! I'm adding the the `otp_phone_label_info` span back and removing the logic to check for it from the JS.

Reply:
Nice. That fixed the Code Climate issues. This should hopefully be good to go now.